excepthook.py

```diff
@@ -14,15 +14,23 @@ def uncaught_exception(exctype, value, tb):
     seconds = delta.total_seconds()
     tr = '\n'.join((traceback.format_tb(tb)))
     exception_only = ''.join(traceback.format_exception_only(exctype, value)).strip()
-    logged_msg = exception_only + '\n' + str(now) + " UTC" + '\n' + tr + '\n\n'
```

Comment:
I think it'll look better if this is all on one line, but keep using format/named parameters